pkgs/applications/video/aegisub/default.nix

```diff
@@ -37,64 +39,39 @@
 
 , portaudioSupport ? false
 , portaudio ? null
+
 }:
 
 assert spellcheckSupport -> (hunspell != null);
-assert automationSupport -> (lua != null);
+assert automationSupport -> (luajit != null);
 assert openalSupport -> (openal != null);
 assert alsaSupport -> (alsa-lib != null);
 assert pulseaudioSupport -> (libpulseaudio != null);
 assert portaudioSupport -> (portaudio != null);
 
 let
+  luajit52 = luajit.override { enable52Compat = true; };
   inherit (lib) optional;
 in
 stdenv.mkDerivation rec {
   pname = "aegisub";
-  version = "3.2.2";
+  version = "3.3.2.20220513";
```

Comment:
Why the cryptic version number?

Reply:
I figured it made sense given that's the date of the Git commit being used. However, looking over my changes after reading this comment made me realize that using the latest Git actually was unnecessary; it's back to 3.3.2 (with the `no-git.patch` applied) now.